transport/grpc.go

```diff
@@ -133,10 +135,29 @@ func (t *grpcTransport) Call(ctx context.Context, request *Request) (*Response,
 	return yarpcResponseToResponse(transportResponse)
 }
 
+func (t *grpcTransport) CallStream(ctx context.Context, request *StreamRequest) (*transport.ClientStream, error) {
+	return t.StreamOutbound.CallStream(ctx, t.requestToYARPCStreamRequest(request))
+}
+
 func (t *grpcTransport) Close() error {
 	return multierr.Combine(t.Transport.Stop(), t.Outbound.Stop())
 }
 
+func (t *grpcTransport) requestToYARPCStreamRequest(streamRequest *StreamRequest) *transport.StreamRequest {
```

Comment:
seeing a bunch of `streamRequest.Request.Field` here. Do we need to check nil on `streamRequest.Request` before we use it? Actually we might even need to check nil on `streamRequest` itself as well?